documentation/modules/uninstalling-mtv-cli.adoc

```diff
@@ -6,31 +6,54 @@
 [id="uninstalling-mtv-cli_{context}"]
 = Uninstalling {project-short} from the command line interface
 
-You can uninstall {project-first} from the command line interface (CLI) by deleting the +{namespace}+ project and the `forklift.konveyor.io` custom resource definitions (CRDs).
+You can uninstall {project-first} from the command line interface (CLI).
 
 .Prerequisites
 
 * You must be logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Delete the project:
+. Delete the `forklift` controller by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} delete project {namespace}
+$ oc delete ForkliftController --all -n openshift-mtv
 ----
 
-. Delete the CRDs:
+. Delete the {project-short} Operator by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc delete Subscription mtv-operator -n openshift-mtv
+----
+
+. Delete the plugin console CR by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc delete ConsolePlugin forklift-console-plugin
+----
+
+. Delete the custom resource definitions by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ {oc} get crd -o name | grep 'forklift' | xargs {oc} delete
 ----
 
-. Delete the OAuthClient:
+. Optional: Delete the project by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} delete oauthclient/forklift-ui
+$ oc delete project <namespace>
```

Comment:
Deleting the project is a dangerous operation in k8s, it's should be done only if the user knows what they are doing.

An example of issue is deleting a pod that allow finalizing of other resources, deleting a namespace with such a pod may create a broken k8s namespace.